urid/src/mapper.rs

```diff
@@ -40,6 +64,43 @@ pub trait URIDMapper {
     /// achieve this.
     fn map(&self, uri: &Uri) -> Option<URID>;
 
+    /// Unsafe wrapper of the `map` method, used by the feature interface.
+    unsafe extern "C" fn extern_map(
```

Comment:
Remember putting a function on a trait makes it public, callable and override-able! You have to put this method separate from the trait! (See my comment on the PR)

Reply:
I actually see no problem with that, as stated before!